backend/src/controllers/image-controller.ts

```diff
@@ -0,0 +1,206 @@
+import { Request, Response } from 'express'; 
+import { validationResult } from 'express-validator';
+import Image from '../models/image';
+import Item from '../models/item'; 
+import mongoose from 'mongoose';
+
+export const createImage = async (req: Request, res: Response) => {
+    // Check for validation errors 
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+        console.log("errors", errors);
+        return res.status(400).json({ errors: errors.array() });
+    }
+ 
+    function isEmpty(arg: any){
```

Comment:
Don't define functions in functions
Also, remove the `isEmpty` function and use `express-validator` to catch the errors (see the `auth-routes` or `dummy-routes` file, for instance)